packages/commerce-server/src/format-prices-for-product.ts

```diff
@@ -194,10 +194,14 @@ export async function formatPricesForProduct(
   const bulkDiscountAvailable =
     bulkCouponPercent > 0 && appliedMerchantCouponLessThanBulk && !pppApplied
 
+  const unitPrice: number = price.unitAmount.toNumber()
+  const fullPrice: number = unitPrice * quantity - fixedDiscountForUpgrade
+
   let defaultPriceProduct: FormattedPrice = {
     ...product,
     quantity,
-    unitPrice: price.unitAmount.toNumber(),
+    unitPrice,
+    fullPrice,
```

Comment:
Compute `fullPrice` in the commerce-server, so that anyone getting a `FormattedPrice` object can make use of that value already computed server-side and not have to recompute these values.

Open to suggestions on the name, `fullPrice` seemed to be how downstream components were thinking of it, but there may be better options for what to call this.